jazzhands/__main__.py

```diff
@@ -196,18 +207,24 @@ def main(argv):
 
         if index_files['js'] and js_dir:
             jsx_registry_path = os.path.join(os.path.dirname(index_files['js']), 'jsx_registry.js')
-            subprocess.call(['python', 'manage.py', 'compilejsx'], stdout=open(jsx_registry_path, 'w'))
-            process_jsx(jsx_registry_path)
-            
+            try:
+                import django_jsx
+                subprocess.call(['python', 'manage.py', 'compilejsx'], stdout=open(jsx_registry_path, 'w'))
+                process_jsx(jsx_registry_path)
+            except ImportError:
+                pass
             collect_app_asset_src(app_asset_dirs, 'js')
 
     ### BUILD
 
     if DO_BUILD:
+        
+        # Build a bundle from the LESS, if present, then append the Stylus bundle, if present.
+        os.unlink(os.path.join(css_dir, 'bundle.css'))
+        if index_files.get('less') and css_dir:
+            build_less(app_asset_dirs)
         if index_files.get('styl') and css_dir:
```

Comment:
If we've found less or stylus files, but haven't figured out where to put the compiled result file, that seems like it should be a fatal error with a helpful message.  (and similarly for js files)

Reply:
Good point!